docs/serverless/08-04-trigger-function-with-event.md

```diff
@@ -72,8 +72,8 @@ These variables refer to the following:
 
     </details>
     <details>
-    <summary label="console-ui">
-    Console UI
+    <summary label="ui">
+    UI
```

Comment:
```suggestion
    Console UI
```
Refer to [this comment](https://github.com/kyma-project/kyma/pull/9296#discussion_r478304363).